src/utils/functions.test.js

```diff
@@ -0,0 +1,77 @@
+import { getStrengthColor, getConfigValue, getStrengthText } from './functions'
+import { STRENGTH_COLOR, STRENGTH_TEXT } from '../constants'
+
+describe('functions', () => {
+
+  describe('functions.getConfigValue', () => {
+    const DEFAULT_CONFIG = { 1: 'first default option', 2: 'second default option' }
+    const CUSTOM_CONFIG = { 1: 'first custom option', 2: 'second custom option' }
+
+    test('should return value from default config', () => {
+      const score = 1
+      const defaultValue = getConfigValue(DEFAULT_CONFIG)(score, undefined)
+
+      expect(defaultValue).toBe(DEFAULT_CONFIG[score])
+    })
+
+    test('should return value from custom config', () => {
+      const score = 1
+      const defaultValue = getConfigValue(DEFAULT_CONFIG)(score, CUSTOM_CONFIG)
+
+      expect(defaultValue).toBe(CUSTOM_CONFIG[score])
+    })
+
+
+  })
+
+  describe('functions.getStrenghColor', () => {
+
+    const CUSTOM_COLORS = {
+      0: 'red',
+      0: 'green',
+      0: 'pink',
+      0: 'blue',
```

Comment:
That's right?

For example, here shouldn't be:
```
  0: 'red',
  1: 'green',
  2: 'pink',
  3: 'blue',
```

instead of all zeros?

The test's right and passes the same way, but because of this is little confusing, don't you think?